lib/reverse_sentence.rb

```diff
@@ -1,4 +1,48 @@
 # A method to reverse the words in a sentence, in place.
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  return nil if my_sentence == nil
+  return my_sentence if (my_sentence.length == 0 || my_sentence.length == 1)
```

Comment:
Since all the string manipulation are happening in-place, you don't need to return anything. Lines 3 & 4 could simply be:
```ruby
  return if my_sentence == nil
  return if my_sentence.length < 2
```